src/vi_engine.rs

```diff
@@ -43,8 +43,8 @@ impl ViEngine {
                         _ => {}
                     },
                     (Some(partial), c) => {
-                        match partial {
-                            'd' => match c {
```

Comment:
This lint is indicating that we need to implement more commands (Most likely through the implementation in `src/vi_parser.rs`)

Reply:
Let me know if I have to remove this, thanks.

Reply:
JT started working on a more systematic parser with the last stream(s) in the other and I put my additions there. So this is not really an issue if it becomes superseded.